examples/loki-stack/role.yaml

```
@@ -17,15 +17,15 @@ rules:
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name: lokistack-netobserv-tenant-logs
+  name: loki-netobserv-tenant
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
-  name: lokistack-netobserv-tenant-logs
+  name: loki-netobserv-tenant
 subjects:
 - kind: ServiceAccount
   name: flowlogs-pipeline
-  namespace: network-observability
+  namespace: netobserv
 - kind: ServiceAccount
   name: network-observability-plugin
```

this guy has been renamed 
```suggestion
  name: netobserv-plugin
```